hazelcast/src/main/java/com/hazelcast/query/Predicates.java

```diff
@@ -29,77 +29,342 @@
 import com.hazelcast.query.impl.predicates.OrPredicate;
 import com.hazelcast.query.impl.predicates.RegexPredicate;
 
+import java.util.Date;
+
 /**
  * A utility class to create {@link com.hazelcast.query.Predicate} instances.
+ * <p>
+ * <b>Special Attributes</b>
+ * <p>
+ * The predicate creation methods accepting an attribute name support two special attributes:
+ * <ul>
+ * <li>{@code "__key"} - instructs the predicate to act on a key associated with an item.
+ * <li>{@code "this"} - instructs the predicate to act on a value associated with an item.
```

Comment:
"act on the key"
"act on the value"

There's also an enum constant for these: `QueryConstants`.

Reply:
fixed